components/voilanorbert/voilanorbert.app.mjs

```diff
@@ -1,11 +1,102 @@
+import { axios } from "@pipedream/platform";
+import { objectToString } from "./utils.mjs";
+
 export default {
   type: "app",
   app: "voilanorbert",
-  propDefinitions: {},
+  propDefinitions: {
+    contactId: {
```

Comment:
Can we use `async options` here. It seems [here](https://api.voilanorbert.com/2018-01-08/#contacts-get) that contacts can be fetched